plugins/generator-1.19.2/forge-1.19.2/procedures/block_nbt_num_get.java.ftl

```diff
@@ -1,10 +1,10 @@
 <#include "mcelements.ftl">
 <#-- @formatter:off -->
 (new Object(){
-	public double getValue(LevelAccessor world, BlockPos pos, String tag) {
-		BlockEntity blockEntity=world.getBlockEntity(pos);
-		if(blockEntity != null) return blockEntity.getPersistentData().getDouble(tag);
-		return -1;
+	public double getValue(LevelAccessor _world, BlockPos _pos, String _tag) {
+		BlockEntity _blockEntity = _world.getBlockEntity(_pos);
+		if(_blockEntity != null) return _blockEntity.getPersistentData().getDouble(_tag);
+		return 0;

```

Comment:
Users may expect this to return -1 in existing procedures, so this change can not be done in this PR and is also massively unrelated to goal of this pr (+ changes like those should be announced on the PR list of changes so reviewers don't accidentally miss such things). -1 may not be the best choice, but it is like this now and can't just be changed like this

Reply:
May apply to others, did not go through whole PR yet